### DISCUSSION TITLE
Vaultwarden Patch - Time to Docker Vaultwarden/Server?

### DISCUSSION THREAD

**Discussion by schafferm** [dani-garcia/vaultwarden#2727] [Q&A] [answered]
Hi - Kind of new around here so please, this might the wrong place to ask:

In Vaultwarden issues the team wrote about a patch that fixes the file upload send by the mobile clients.
It fixes https://github.com/dani-garcia/vaultwarden/issues/2644 by always providing a Content-Type even though one
isn't set in this specific case....
Until then, we can use this patched version of multer-rs.

My question is how do I get this patched version or the file into the most current docker or if not - when can would it get updated?

Thanks - Michael

**Comment by BlackDex** [comment, marked as answer]
Hello @schafferm, if you want the version containing this patch you should use the `testing` tagged images. These are based upon the main branch. The `latest` tagged images are triggered upon new releases.

**Reply by schafferm** [reply]
Thank you - that makes sense - was not aware that the patched images would come under testing
